File: week06/mario.py
from cs50 import get_int

# Getting the height from the user and checking that it is between 1 and 8
height = get_int("Height: ")
while (height <= 0 or height >= 9):
    height = get_int("Height: ")

# Printing the pyramid of the height given by the user
for row in range(height):
    for column in range(height - row - 1):
        print(" ", end="")
    for column in range(row + 1):
        print("#", end="")
    print("  ", end="")
    for column in range(row + 1):
        print("#", end="")
    print("")

File: week01/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int spaces;
    int hashs;
    do
    {
        height = get_int("Height: ");
    }
    while (1 > height || height > 8);
    for (int i = 0; i < height; i++)
    {
        for (spaces = 0; spaces < height - i - 1; spaces++)
        {
            printf(" ");
        }
        for (hashs = 0; hashs < i + 1; hashs++)
            printf("#");
        printf("  ");
        for (hashs = 0; hashs < i + 1; hashs++)
            printf("#");
        printf("\n");
    }
}
